lib/internal/errors.js

```diff
@@ -86,3 +86,4 @@ module.exports = exports = {
 // Note: Please try to keep these in alphabetical order
 E('ERR_ASSERTION', (msg) => msg);
 // Add new errors from here...
+E('ERR_SLAVE_CLOSED_BEFORE_REPLY','Slave closed before reply');
```

Comment:
nit: running `make lint` should complain on this. There needs to be a single space after the comma before `'Slave...`